### DISCUSSION TITLE
how to make throttle position audible

### DISCUSSION THREAD

**Discussion by fnijenhuis5** [EdgeTX/edgetx#5356] [General]
is there anyone out there that can help me with making my throttle stick position audible
like throthle 10%, throthle 20%
i use radiomaster tx16s markII with edgetx 2.10 
kind regards 
frans nijenhuis

**Comment by pfeerick** [comment]
I think I saw you  [ask also on RCG](https://www.rcgroups.com/forums/showpost.php?p=52744351&postcount=7873)  and [got this answer here](https://www.rcgroups.com/forums/showpost.php?p=52744557&postcount=7875). I have had this open for a day or two meaning to respond, but hadn't gotten to it, so hopefully you were able to get this working how you wanted. Otherwise, I was going to suggest a Lua script that could sample the stick position, and then do the callout when the stick was in one matching positions.

**Comment by fnijenhuis5** [comment]
i am not familiar with luna scripts so i solved my problem with logical switches that read out the position of the stick a~x 50
than i made the same amount of special functions that plays a track saying 50 %
this works beatifull for me 
thank for thinking with me 
kind regards 
frans nijenhuis

**Comment by fnijenhuis5** [comment]
Thanks , see my reply on github

Op zo 4 aug 2024 07:50 schreef Peter Feerick ***@***.***>:
